android/src/main/java/io/sentry/react/RNSentryModule.java

```diff
@@ -206,6 +225,54 @@ public void fetchNativeAppStart(Promise promise) {
         RNSentryModule.didFetchAppStart = true;
     }
 
+    /**
+     * Returns frames metrics at the current point in time.
+     */
+    @ReactMethod
+    public void fetchNativeFrames(Promise promise) {
+        if (RNSentryModule.frameMetricsAggregator == null) {
+            promise.resolve(null);
+        } else {
+            try {
+                int totalFrames = 0;
+                int slowFrames = 0;
+                int frozenFrames = 0;
+
+                final SparseIntArray[] framesRates = RNSentryModule.frameMetricsAggregator.getMetrics();
+
+                if (framesRates != null) {
+                    final SparseIntArray totalIndexArray = framesRates[FrameMetricsAggregator.TOTAL_INDEX];
+                    if (totalIndexArray != null) {
+                        for (int i = 0; i < totalIndexArray.size(); i++) {
+                            int frameTime = totalIndexArray.keyAt(i);
+                            int numFrames = totalIndexArray.valueAt(i);
+                            totalFrames += numFrames;
+                            // hard coded values, its also in the official android docs and frame metrics API
+                            if (frameTime > 700) {
```

Comment:
let's extract these numbers (700, 16) into constants, and then put the comments about these values there (It'll mean the git blame if these change are easier to parse as well)